### ISSUE TITLE
Need a way to turn off window.opener handling

### ISSUE THREAD

**Issue by cobexer** [tntim96/JSCover#197]
In my CI environment I run Internet Explorer against my application, however if I open another window from within my application the second window merges it's coverage data with the first window. In Internet Explorer that makes the second window execute very slowly and for example causes the call to jscoverage_report to take upwards of 170 Seconds!!(my test automation often kills my browser during this time) In a lengthy process I optained a profiling result that showed around 150 calls to getArrayJSON.

(in my combined coverage I have around 70 files and 25500 statements)

I use JSCover in server mode, have an Apache reverse proxy in front of it (to make use easier in my CI and to avoid sending the webserver requests through JSCover) and manage storing the jscover data myself.

So to fix my performance issue I need a way to turn off window.opener integration.

**Comment by tntim96** [comment]
I think the relevant code is the first 34 lines of [header.js](https://github.com/tntim96/JSCover/blob/master/src/main/resources/header.js). I could add a switch, say `--isolate-browsers`, to turn this off - is that what you need?

**Comment by cobexer** [comment]
Yes if that code in the header.js would be skipped that would indeed solve the problem I am seeing.
I guess that switch would be useful for all modes that do instumentation - that would also help a coworker that used to patch JSCover to avoid the connection.

**Comment by tntim96** [comment]
Before I commit, can you test with either [the test JAR](http://jscover.sourceforge.net/JSCover-all.jar) or the [1.0.20-SNAPSHOT](https://oss.sonatype.org/content/repositories/snapshots/com/github/tntim96/JSCover/1.0.20-SNAPSHOT/) which will need this in your POM
```xml
<pluginRepository>
    <id>Sonatype repository</id>
    <name>Sonatype's Maven repository</name>
    <url>https://oss.sonatype.org/content/repositories/snapshots/</url>
</pluginRepository>
```

The switch to use is `--isolate-browser`.

**Comment by tntim96** [comment]
Actually, can you confirm if you use `jscoverage.html` page at all? If not, I may change my implementation. It doesn't seem to make sense excluding combining coverage when using that page.

**Comment by cobexer** [comment]
I just tried this build in our CI and this solved all issues I had with timeouts / hangs on Internet Explorer!

Thanks for the prompt fix!

**Comment by tntim96** [comment]
Great. This will go in release 1.0.20

**Comment by tntim96** [comment]
I've just released 1.0.20 if you want to use that. Should appear in Maven repository soon.

**Comment by cobexer** [comment]
Deployed 1.0.20 to the CI - works great!
Thanks for the fast fix!
